llvm/include/llvm/LTO/LTOBackend.h

```diff
@@ -56,6 +56,7 @@ Error thinBackend(const Config &C, unsigned Task, AddStreamFn AddStream,
                   const FunctionImporter::ImportMapTy &ImportList,
                   const GVSummaryMapTy &DefinedGlobals,
                   MapVector<StringRef, BitcodeModule> *ModuleMap,
+                  bool CodeGenOnly,
```

Comment:
nit: add a brief comment for `CodeGenOnly` parameter before method `thinBackend`.